### ISSUE TITLE
Nothing happens on Windows

### ISSUE THREAD

**Issue by wastingaeons** [quietcore/TraktForVLC#2]
Hi, first off, thanks for taking this on to try to get scrobbling to trakt with VLC

I'm on Windows 7 64bit. Looking at the log file I get:

2012-11-19 17:22:11,427::VLCRemote::DEBUG::-> Sending: get_title
2012-11-19 17:22:11,428::VLCRemote::DEBUG::<- Recived: Childrens Hospital - 4x07 - British Hospital.mkv
2012-11-19 17:22:13,430::VLCRemote::DEBUG::cleaning cache
2012-11-19 17:22:13,431::VLCRemote::DEBUG::<- Recived: 
2012-11-19 17:22:13,431::VLCRemote::DEBUG::-> Sending: get_length
2012-11-19 17:22:13,431::VLCRemote::DEBUG::<- Recived: 691
2012-11-19 17:22:13,431::VLCRemote::DEBUG::cleaning cache
2012-11-19 17:22:13,431::VLCRemote::DEBUG::<- Recived: 
2012-11-19 17:22:13,431::VLCRemote::DEBUG::-> Sending: get_time
2012-11-19 17:22:13,433::VLCRemote::DEBUG::<- Recived: 5
2012-11-19 17:22:13,433::TraktForVLC::DEBUG::{'episode': '7', 'title': 'Childrens Hospital', 'tv': True, 'season': '4', 'year': 2010, 'duration': 691, 'percentage': 0}
2012-11-19 17:22:13,433::TraktForVLC::DEBUG::Timer: 0

But nothing after that, the episode ends and nothing shows up on the trakt.tv website

**Comment by quietcore** [comment]
I've only tested the code in windows xp. I've not had a chance to test it in windows 7 64x so there could be issues with it working in that environment. I would assume you setup the config file with your trakt username and password if you got this far. 

it's been a while since i've had time to work on this so it's possible that there have been api changes.

**Comment by wastingaeons** [comment]
Correct, I followed the instructions step by step. Is there a way to debug further? To see if we get a response from trakt?
